posthog/utils.py

```diff
@@ -5,6 +5,7 @@
 import gzip
 import hashlib
 import json
+import logging
```

Comment:
Can we use `structlog` as per [docs](https://posthog.com/docs/contribute/coding-conventions#format) ? 🙏

Reply:
Ah thank you for the link!